src/legacy/components/OrderProgressBar/index.tsx

```diff
@@ -92,18 +93,20 @@ export function OrderProgressBar(props: OrderProgressBarProps) {
     trail: 3000,
   })
 
-  useEffect(() => {
+  const updatePercentage = useCallback(() => {
     if (!isPending) {
       return
     }
 
-    const id = setInterval(() => {
-      const percentage = getPercentage(elapsedSeconds, expirationInSeconds, chainId)
-      setPercentage(percentage)
-    }, REFRESH_INTERVAL_MS)
+    const percentage = getPercentage(elapsedSeconds, expirationInSeconds, chainId)
+    setPercentage(percentage)
+  }, [chainId, elapsedSeconds, expirationInSeconds, isPending])
 
-    return () => clearInterval(id)
-  }, [creationTime, validTo, chainId, elapsedSeconds, expirationInSeconds, isPending])
```

Comment:
Anyone knows why `creationTime` was part of the useEffect? 

If we need to trigger it under some field changes we can, but  I'm not sure this is what we want. We already update 5 times per second!

`useEffect(updatePercentage, [creationTime, validTo, updatePercentage])`

Reply:
Can you use git blame and check when it's added?
At first glance I'd guess it was used in the `useEffect` and later removed, but the deps array was not updated